src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -569,19 +569,109 @@ class FeatureSearchStore implements IFeatureSearchStore {
     }
 }
 
+const applyStaleConditions = (
+    query: Knex.QueryBuilder,
+    staleConditions?: IQueryParam,
+): void => {
+    if (!staleConditions) return;
+
+    const { values, operator } = staleConditions;
+
+    if (!values.includes('potentiallyStale')) {
+        applyGenericQueryParams(query, [
+            {
+                ...staleConditions,
+                values: staleConditions.values.map((value) =>
+                    value === 'active' ? 'false' : 'true',
+                ),
+            },
+        ]);
+        return;
+    }
+
+    if (values.length === 3) {
+        switch (operator) {
+            case 'IS':
+            case 'IS_ANY_OF':
+                // All flags included; no action needed
+                break;
+            case 'IS_NOT':
+            case 'IS_NONE_OF':
+                // All flags excluded
+                query.whereNotIn('features.stale', [false, true]);
+                break;
+        }
+        return;
+    }
+
+    if (values.length === 1) {
```

Comment:
Are you thinking that values might be `['stale', 'stale', 'stale']`?

Reply:
We can change the handling to check explicitly for all three states, though, I don't mind that.

Reply:
Or just turn it into a set

Reply:
Changed it into a set in a975b8d272e660e96b97f9cc7847f4e1e7f61619. Does that address your concern? Otherwise, I'm not sure I understand what you're asking about.